starknet_py/transactions/deploy.py

```diff
@@ -0,0 +1,25 @@
+from typing import Union, List, Optional
+
+from starkware.starknet.definitions.fields import ContractAddressSalt
+
+from starknet_py.net.client_models import ContractClass
+from starknet_py.net.models import Deploy
+
+
+def make_deploy_tx(
+        compiled_contract: Union[ContractClass, str],
+        constructor_calldata: List[int],
+        salt: Optional[int] = None,
+        version: int = 0,
```

Comment:
Perhaps we could also add option to pass `compilation source` as in declare?

Reply:
ah i don't quite like it because it adds too many arguments to the function, but if you really want i'll do it

Reply:
These are helper methods so I think it's fine if they're like this

Reply:
These are helper methods so I think it's fine if they're like this